stubs/stripe/stripe/api_resources/abstract/searchable_api_resource.pyi

```diff
@@ -0,0 +1,20 @@
+from _typeshed import Self
+from collections.abc import Iterator
+
+from stripe.api_resources.abstract.api_resource import APIResource as APIResource
+from stripe.api_resources.search_result_object import SearchResultObject
+
+class SearchableAPIResource(APIResource):
+    @classmethod
+    def _search(
+        cls: type[Self],
+        search_url: str,
+        api_key: str | None = ...,
+        stripe_version: str | None = ...,
+        stripe_account: str | None = ...,
+        **params,
+    ) -> SearchResultObject[Self]: ...
```

Comment:
Hmm, is this method public-API? We usually prefer to omit private methods (unless there's a good reason to make an exception) -- including them increases our maintenance burden, and users _probably_ shouldn't be using them anyway :)

Reply:
No this is private. Got it, will remove, thanks!